templates/PreviewDataObject.ss

```diff
@@ -5,15 +5,19 @@
   </head>
   <body>
     <div class="content fw-typography-spacing content-typography-space">
-
       <% if $Rendered %>
         $Rendered
       <% else %>
         <div style="text-align: center;">
           Object not found or not created yet.
         </div>
       <% end_if %>
-
     </div>
+    <% if $ID %>
+        <script>
+            var element = document.querySelector("[data-id='$ID']");
+            element.scrollIntoView();
```

Comment:
I almost think that this should show some border effect as well. Maybe something that only shows temporarily. Let me know your thoughts.

Reply:
@tiller1010 I thought about this while doing it, but I am not a fan of having a border around the content - just not pleasing on the eye and there is possibility of the border looking like its part of the actual theme/content.

I actually played around with adding opacity to the other elements, so the content you were editing was more apparent - but it did not work well in my opinion. I was thinking of handling it separately in a separate task after the initial functionality of it being in view is complete.

Reply:
@tiller1010 I added a box-shadow to the preview window, so everything should be darkened out expect for the content you are editing. Can you take a look and provide some feedback? Not sure if its too much or if it is helpful -let me know your thoughts

Reply:
This worked good for me. I can only see this being a potential issue with darker backgrounds

Reply:
hmm ya good point, we can test it out and adjusts as needed I think. I think Katie and Alyssa's feedback will be useful here!